ibm_catalog.json

```diff
@@ -1,428 +1,474 @@
-                  },
-                  {
-                    "displayname": "direct",
-                    "value": "direct"
-                  }
-                ]
-              },
-              {
-                "key": "existing_cos_instance_crn"
-              },
-              {
-                "key": "existing_log_archive_cos_bucket_name"
-              },
+  "products": [
+    {
+      "name": "deploy-arch-ibm-observability",
+      "label": "Observability",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "logging_monitoring",
+        "solution"
+      ],
+      "keywords": [
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution",
+        "observability",
+        "observability instances",
+        "log analysis",
+        "cloud monitoring",
+        "ATracker",
+        "Activity Tracker",
+        "observability agents"
+      ],
+      "short_description": "Configures IBM Cloud Observability resources",
+      "long_description": "Solutions which support configuring IBM Cloud Observability resources for logging, monitoring and activity tracking.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-observability-da/main/images/observability-icon.svg",
+      "provider_name": "IBM",
+      "features": [
+        {
+          "title": "Configures Log Analysis",
+          "description": "Configures Log Analysis instance and agents."
+        },
+        {
+          "description": "Configures Cloud Monitoring",
+          "title": "Configures Cloud Monitoring. instance and agents."
+        },
+        {
+          "description": "Configures Activity Tracker routing",
+          "title": "Configures an Activity Tracker route with a COS bucket target."
+        }
+      ],
+      "flavors": [
+        {
+          "label": "Instances",
+          "name": "instances",
+          "install_type": "fullstack",
+          "working_directory": "solutions/instances",
+          "iam_permissions": [
+            {
+              "service_name": "atracker",
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Writer",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ]
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
+              "service_name": "logdna"
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
+              "service_name": "hs-crypto"
```

Comment:
I guess `hs-crypto` makes sense if the DA is configured to create a HPCS root key (which is supported), however I would also expect to see `kms` to cover Key Protect here too

Reply:
Added